### PR TITLE
(tlg4019)(tlg4026)(tlg0565)(tlg2048)(tlg0250)(tlg2057)_.cts_xml_and_headers

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated cts_.xml and the headers to current standards.

### CODE REVIEW

File: data/tlg4019/tlg001/tlg4019.tlg001.opp-grc1.xml [hunk 1]
```diff
@@ -64,20 +64,19 @@
         <listBibl>
           <biblStruct>
             <monogr>
-              <title xml:lang="lat">Olympiodori prolegomena et in categorias commentarium </title>
-              <editor>
-                <persName>
```
Comment (lcerrato) [inline data/tlg4019/tlg001/tlg4019.tlg001.opp-grc1.xml:69]: The `<persName>` tag usage isn't incorrect — it's just hit or miss whether it appears or not. There probably should be a uniform standard.

File: data/tlg4019/tlg002/tlg4019.tlg002.opp-grc1.xml [hunk 2]
```diff
@@ -64,20 +64,19 @@
...
              <author ref="urn:cts:greekLit:tlg4019" xml:lang="lat">Olympiodorus</author>
               <imprint>
                 <publisher>Reimer</publisher>
                 <pubPlace>Berlin</pubPlace>
                 <date>1902</date>
               </imprint>
-        <biblScope unit="volume">12.1</biblScope>
             </monogr>
+            <series>
+              <title>Commentaria in Aristotelem graeca</title>
```
Comment (lcerrato) [inline data/tlg4019/tlg002/tlg4019.tlg002.opp-grc1.xml:77]: title is in Latin

File: data/tlg0565/tlg001/tlg0565.tlg001.1st1K-grc1.xml [hunk 3]
```diff
@@ -50,20 +49,20 @@
...
  <biblStruct>
  <monogr>
-  <title>Sorani Gynaeciorum Libri IV De signis Fracturarum De Fasciis Vita Hippocratis Secundum Soranum</title>
-  <editor>
-	<persName>
-	 <name>Johannes Ilberg</name>
-	</persName>
-  </editor>
+  <title xml:lang="lat">Sorani Gynaeciorum Libri IV</title>
+  <title type="sub">De Signis Fracturarum De Fasciis Vita Hippocratis Secundum Soranum</title>
```
Comment (lcerrato) [inline data/tlg0565/tlg001/tlg0565.tlg001.1st1K-grc1.xml:53]: `<title type="sub">` needs language attribute

File: data/tlg0565/tlg001/tlg0565.tlg001.1st1K-grc1.xml [hunk 4]
```diff
@@ -50,20 +49,20 @@
...
   <author ref="urn:cts:greekLit:tlg0565">Soranus</author>
   <imprint>
 	<publisher>Teubner</publisher>
 	<pubPlace>Leipzig</pubPlace>
 	<date>1927</date>
   </imprint>
-  <biblScope unit="volume">4</biblScope>
  </monogr>
+  <series>
+   <title>Corpus Medicorum Graecorum</title>
```
Comment (lcerrato) [inline data/tlg0565/tlg001/tlg0565.tlg001.1st1K-grc1.xml:63]: `<title>` needs language attribute

File: data/tlg0565/tlg002/__cts__.xml [hunk 5]
```diff
@@ -2,6 +2,7 @@
   <ti:title xml:lang="lat">De signis Fracturarum</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg0565.tlg002.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0565.tlg002">
     <ti:label xml:lang="lat">De signis Fracturarum</ti:label>
-    <ti:description xml:lang="lat">Soranus, De signis Fracturarum, Ilberg, Teubner, 1927</ti:description>
+    <ti:description xml:lang="lat">Soranus. Sorani Gynaeciorum Libri IV: De Signis Fracturarum De Fasciis Vita Hippocratis Secundum Soranum. Ilberg, Johannes, editor.
```
Comment (lcerrato) [inline data/tlg0565/tlg002/__cts__.xml:5]: description language should be "mul"

File: data/tlg0565/tlg001/__cts__.xml [hunk 6]
```diff
@@ -1,7 +1,8 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0565" xml:lang="grc" urn="urn:cts:greekLit:tlg0565.tlg001">
   <ti:title xml:lang="lat">Gynaeciorum Libri IV</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg0565.tlg001.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0565.tlg001">
-    <ti:label xml:lang="lat">Sorani Gynaeciorum Libri IV</ti:label>
-    <ti:description xml:lang="lat">Soranus, Gynaeciorum, Ilberg, Teubner, 1927</ti:description>
+    <ti:label xml:lang="lat">Gynaeciorum Libri IV</ti:label>
+    <ti:description xml:lang="lat">Soranus. Sorani Gynaeciorum Libri IV: De Signis Fracturarum De Fasciis Vita Hippocratis Secundum Soranum. Ilberg, Johannes, editor.
```
Comment (lcerrato) [inline data/tlg0565/tlg001/__cts__.xml:5]: description language should be "mul"

File: data/tlg0565/tlg002/tlg0565.tlg002.1st1K-grc1.xml [hunk 7]
```diff
@@ -49,21 +48,21 @@
...
-  <imprint>
-	<publisher>Teubner</publisher>
-	<pubPlace>Leipzig</pubPlace>
-	<date>1927</date>
-  </imprint>
-  <biblScope unit="volume">4</biblScope>
- </monogr>
+  <monogr>
+   <title xml:lang="lat">Sorani Gynaeciorum Libri IV</title>
+   <title type="sub">De Signis Fracturarum De Fasciis Vita Hippocratis Secundum Soranum</title>
```
Comment (lcerrato) [inline data/tlg0565/tlg002/tlg0565.tlg002.1st1K-grc1.xml:53]: <title type="sub"> should have a language attribute

File: data/tlg0565/tlg002/tlg0565.tlg002.1st1K-grc1.xml [hunk 8]
```diff
@@ -49,21 +48,21 @@
...
+   <editor>Johannes Ilberg</editor>
+   <author ref="urn:cts:greekLit:tlg0565">Soranus</author>
+   <imprint>
+    <publisher>Teubner</publisher>
+    <pubPlace>Leipzig</pubPlace>
+    <date>1927</date>
+   </imprint>
+  </monogr>
+  <series>
+   <title>Corpus Medicorum Graecorum</title>
```
Comment (lcerrato) [inline data/tlg0565/tlg002/tlg0565.tlg002.1st1K-grc1.xml:63]: <title> should have a language attribute

File: data/tlg0565/tlg003/__cts__.xml [hunk 9]
```diff
@@ -2,6 +2,7 @@
   <ti:title xml:lang="lat">De Fasciis</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg0565.tlg003.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0565.tlg003">
     <ti:label xml:lang="lat">De Fasciis</ti:label>
-    <ti:description xml:lang="lat">Soranus, De Fasciis, Ilberg, Teubner, 1927</ti:description>
+    <ti:description xml:lang="lat">Soranus. Sorani Gynaeciorum Libri IV: De Signis Fracturarum De Fasciis Vita Hippocratis Secundum Soranum. Ilberg, Johannes, editor.
```
Comment (lcerrato) [inline data/tlg0565/tlg003/__cts__.xml:5]: <ti:description xml:lang="lat"> language should be mul

File: data/tlg0565/tlg003/tlg0565.tlg003.1st1K-grc1.xml [hunk 10]
```diff
@@ -49,21 +48,21 @@
...
-  <imprint>
-	<publisher>Teubner</publisher>
-	<pubPlace>Leipzig</pubPlace>
-	<date>1927</date>
-  </imprint>
-  <biblScope unit="volume">4</biblScope>
- </monogr>
+  <monogr>
+   <title xml:lang="lat">Sorani Gynaeciorum Libri IV</title>
+   <title type="sub">De Signis Fracturarum De Fasciis Vita Hippocratis Secundum Soranum</title>
```
Comment (lcerrato) [inline data/tlg0565/tlg003/tlg0565.tlg003.1st1K-grc1.xml:53]: <title type="sub"> needs language attribute

**Review by lcerrato — CHANGES_REQUESTED** [review]
I started to go file by file — this is a big PR — but the same themes keep popping up. Might be worth it to just look at the <title type="sub"> and the cts description language for all files?

File: data/tlg0565/tlg003/tlg0565.tlg003.1st1K-grc1.xml (hunk 10 above)
Comment (AlisonBabeu) [reply data/tlg0565/tlg003/tlg0565.tlg003.1st1K-grc1.xml:53]: Hey @lcerrato thanks for reviewing and I will make these changes. And this pull request as well as the other one you mentioned (tlg5075) are all on the same branch, I had created sub branches for them and actually did five or six different pull requests thinking somehow that would not create this mega pull request. Sorry. From now I'm just going to create a branch per textgroup even if it means there are more pull requests as they will be much easier to look at. Sorry for the all the work.

**Comment by lcerrato** [comment]
@AlisonBabeu 
I would say 10 or fewer files per branch? I don't think these are sub-branches, though. If the PRs were all on related branches, they would update to the same pull request. Instead, you have a series of pull requests that sometimes overlap, but sometimes don't.
https://github.com/OpenGreekAndLatin/First1KGreek/pull/2465 has identical files in there, such as the tlg4019 and tlg0565 sets.
That means after this is merged, you need to pull from master and revisit that PR, otherwise there will be conflicts and/or changes in that PR that overwrite these changes.

**Review by lcerrato — APPROVED** [review]

**Pull request merged**